Configuration/StandardSequences/python/SimL1Emulator_cff.py

```diff
@@ -11,3 +11,12 @@
     loadTriggerDigiAliases()
     from FastSimulation.Configuration.DigiAliases_cff import gtDigis,gmtDigis,gctDigis,caloStage1LegacyFormatDigis
     
+#HGCal
+def _modifyL1SimForHGCal( theProcess ):
+    theProcess.simEcalTriggerPrimitiveDigis.BarrelOnly = cms.bool(True)  
+
+from Configuration.StandardSequences.Eras import eras
+modifyL1SimForHGCal_ = eras.phase2_hgcal.makeProcessModifier( _modifyL1SimForHGCal )
```

Comment:
This shouldn't be a process modifier and instead be moved to where `simEcalTriggerPrimitiveDigis` is defined and be applied to the object directly via `eras.phase2_hgcal`.